frontend/src/pages/workspace/ui/workspace-workflows-page.module.css

```
@@ -0,0 +1,150 @@
+
+.workflowMeta {
+  display: flex;
+  align-items: center;
+  gap: 10px;
+  flex-wrap: wrap;
+}
+
+.workflowPack,
+.workflowVersion,
+.workflowCode {
+  font-family: "Geist Mono", "SFMono-Regular", monospace;
```

Stylelint (17.9.0)

[error] Expected no quotes around "SFMono-Regular" (font-family-name-quotes)

(font-family-name-quotes)